dockerfiles/ubuntu/is-analytics/dashboard/Dockerfile

```diff
@@ -16,10 +16,43 @@
 #
 # ------------------------------------------------------------------------
 
-# set base Docker image to AdoptOpenJDK Ubuntu Docker image
-FROM adoptopenjdk:8u242-b08-jdk-hotspot-bionic
+# set base Docker image to JDK Ubuntu 20.04 Docker image
+FROM ubuntu:20.04
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
-      com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.7.0.4"
+      com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.7.0.5"
+
+#Install Open JDK
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends tzdata curl ca-certificates fontconfig locales \
+  && echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen \
+  && locale-gen en_US.UTF-8 \
+  && rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_VERSION jdk8u292-b10
```

Comment:
Isn't the JAVA_VERSION value same as the ones we are downloading later `jdk8u322-b06 `

Reply:
Fixed from https://github.com/wso2/docker-is/pull/360/commits/f59fbfbe1d17ce24dbb89263706d00b3720ec4b2